### ISSUE TITLE
Import returns undefined

### ISSUE THREAD

**Issue by gretzke** [ethereumjs/ethereumjs-wallet#144]
When importing the Wallet library in a plain nodeJS application like this
`const { Wallet } = require('ethereumjs-wallet')`
Wallet is undefined.

Workaround: declare `"type": "module"` in `package.json` and import like this:
```
import ethereumjs from 'ethereumjs-wallet';
const Wallet = ethereumjs.default
```

**Comment by ryanio** [comment]
ah it looks like the readme might need to be fixed.

this should work out of the box:

`const Wallet = require('ethereumjs-wallet').default`

**Comment by holgerd77** [comment]
Ah, just had a look in the [code](https://github.com/ethereumjs/ethereumjs-wallet/blob/master/src/index.ts) to get a confirmation that `Wallet` is indeed not exported, I think we should rather tweak the code here to also export `Wallet` as a named export (and optimally remove `default`, but this would be a breaking change)?

Will reopen for now (also for the `README` change which is rather important since this is the central `README` code example).

**Comment by ryanio** [comment]
changing the export from default to named would indeed would be breaking, although we could keep the default and additionally export it as named. seems valid as per [this article](https://www.bignerdranch.com/blog/default-exports-or-named-exports-why-not-both/).

@holgerd77 let me know if you would like for me to open a PR to fix the readme or add the named Wallet export.

**Comment by holgerd77** [comment]
@ryanio yes, if you can go along would be nice if you can open a short PR here. My tendency is to add this as a named export, keep the `default` for now. In contrast to the article I would rather mark the default as deprecated and remove on next major release, I am not completely going along with these "it's convenient to have both" arguments but would rather think that not having an unambiguous and unique way is rather leading to more potential problems (e.g. people using both ways and forgetting the one variant on updates or refactors).

But - despite the large text 😜  - I have absolutely no strong opinions here, and could live with all three ways here (1. only adding named, 2. named + deprecation, 3. README), since all mark an improvement on the status quo. So if you have got additional doubts on one of the variants feel free to choose another.

**Comment by ryanio** [comment]
@holgerd77 great, I opened a PR.

I agree on the deprecation of the default export, but I don't know of a good way of expressing it to the user when actually *using* the library. So we can add a comment above the default export in the source noting that it is deprecated, but I thought that might look a little confusing, so I decided to omit it for now. But in review of the PR if you would like to suggest a change to add a commit for it we can do that.
